.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These files should be removed - I think if you rebase the current `main` on to yours it should pull in an updated gitignore with `.swiftpm` - if not could you add it and remove these?